hana/install.sls

```diff
@@ -2,6 +2,11 @@
 {% set host = grains['host'] %}
 
 {% for node in hana.nodes %}
+{% if node.host == host and node.scenario_type is defined and node.scenario_type.lower() == 'cost-optimized' %}
+include_cost_optimized:
```

Comment:
Is this working? I didn't know we could use *include* this way.

Anyway, maybe, we should consider just having an initial if with **node.host == host** for everything and then other if statements for *cost-optimized* and *install*. What do you think?

Reply:
Hi @arbulu89 I want to test the include this way using an id. The reason because I was getting error with my deployments last time " domain[1] found conflicting ID 'include' "

similar to salt issue https://github.com/saltstack/salt/issues/9400
There exists include already in 'enable_secondary.sls''

Below is relevant error:
-------------------------------------------------------------------------------------------

module.hana_node.module.hana_node.libvirt_domain.domain[0] (remote-exec): [INFO    ] Fetching file from saltenv 'base', ** done ** 'hana/enable_primary.sls'
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec): [INFO    ] Fetching file from saltenv 'base', ** done ** 'hana/install.sls'
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec): [CRITICAL] Rendering SLS 'base:hana.install' failed: while constructing a mapping
module.hana_node.module.hana_node.libvirt_domain.domain[0] (remote-exec): [INFO    ] Fetching file from saltenv 'base', ** done ** 'hana/enable_secondary.sls'
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec):   in "<unicode string>", line 8, column 1:
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec):     include:
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec):     ^
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec): found conflicting ID 'include'
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec):   in "<unicode string>", line 32, column 1:
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec):     include:
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec):     ^
module.hana_node.module.hana_node.libvirt_domain.domain[1] (remote-exec): [INFO    ] Fetching file from saltenv 'base', ** done ** 'hana/enable_primary.sls'

Reply:
The issue was resolved after moving include out of loop